libraries/AP_Vehicle/AP_Vehicle.cpp

```diff
@@ -178,6 +180,11 @@ void AP_Vehicle::loop()
         done_safety_init = true;
         BoardConfig.init_safety();
     }
+    if(!(_last_internal_errors == AP::internalerror().errors())) {
+        AP::logger().Write_InternalError();
+        send_internal_error_statustext(AP::internalerror().errors());
+        _last_internal_errors = AP::internalerror().errors();
+    }
```

Comment:
```suggestion
    const uint32_t new_internal_errors = AP::internalerror().errors();
    if(_last_internal_errors != new_internal_errors) {
        AP::logger().Write_Error(LogErrorSubsystem::INTERNAL_ERROR, LogErrorCode::INTERNAL_ERRORS_DETECTED);
        gcs().send_text(MAV_SEVERITY_CRITICAL, "Internal Errors %x", (unsigned)new_internal_errors);
        _last_internal_errors = new_internal_errors;
    }
```

(avoid calling method multiple times, simplify)